spec/game_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'game'
+
+describe Game do
+  subject(:game) { described_class.new }
+  let(:board) { double :board }
+
+  describe '#initialize' do
+    it 'generates a new board' do
+      expect(game.board).not_to be_nil
```

Comment:
Try to test behaviour, not state: `board` is the state of `game`, not its behaviour.